src/components/sbb-image/sbb-image.tsx

```diff
@@ -409,84 +413,79 @@ export class SbbImage implements ComponentInterface {
 
     const pictureSizeConfigs = this._preparePictureSizeConfigs();
 
-    return (
+    return html`
       <figure
-        class={{
+        class=${classMap({
           image__figure: true,
           [`image__figure--teaser-hero`]: this._variantTeaserHero,
-          [`image__figure--no-radius`]: !this.borderRadius || this._variantTeaserHero,
+          [`image__figure--no-radius`]: this.noBorderRadius || this._variantTeaserHero,
           [`image__figure--ratio-${this.aspectRatio}`]: true,
           [`image__figure--loaded`]: this._loaded,
-        }}
-        {...attributes}
+        })}
+        ${spread(attributes)}
       >
         <div class="image__wrapper">
-          {this.lqip ? (
-            <img
-              alt=""
-              class="image__blur-hash"
-              src={imageUrlLQIP}
-              width="1000"
-              height="562"
-              loading={this.loading}
-              decoding={this.decoding}
-            />
-          ) : (
-            ''
-          )}
+          ${!this.skipLqip
+            ? html`<img
+                alt=""
+                class="image__blur-hash"
+                src=${imageUrlLQIP}
+                width="1000"
+                height="562"
+                loading=${this.loading}
+                decoding=${this.decoding}
+              />`
+            : nothing}
 
           <picture>
-            {/* render picture element sources */}
-            {pictureSizeConfigs.map((config) => {
+            <!-- render picture element sources -->
+            ${pictureSizeConfigs.map((config) => {
               const imageHeight = config.image.height;
               const imageWidth = config.image.width;
               const mediaQuery = this._createMediaQueryString(config.mediaQueries);
-
               return [
-                <source
-                  media={`${mediaQuery}`}
-                  sizes={`${imageWidth}px`}
-                  srcSet={
-                    `${imageUrlWithParams}&w=${imageWidth}&h=${imageHeight}&q=${this._config.nonRetinaQuality} ${imageWidth}w, ` +
-                    `${imageUrlWithParams}&w=${imageWidth * 2}&h=${imageHeight * 2}&q=${
-                      this._config.retinaQuality
-                    } ${imageWidth * 2}w`
-                  }
-                />,
+                html` <source
+                  media=${`${mediaQuery}`}
+                  sizes=${`${imageWidth}px`}
+                  srcset=${`${imageUrlWithParams}&w=${imageWidth}&h=${imageHeight}&q=${this._config.nonRetinaQuality} ${imageWidth}w, ` +
+                  `${imageUrlWithParams}&w=${imageWidth * 2}&h=${imageHeight * 2}&q=${
+                    this._config.retinaQuality
+                  } ${imageWidth * 2}w`}
+                />`,
               ];
             })}
             <img
-              alt={this.alt}
+              alt=${this.alt}
```

Comment:
I think the `|| nothing` is necessary here
```suggestion
              alt=${this.alt || nothing}
```